src/CitizenProject/CitizenProjectManager.php

```diff
@@ -30,12 +33,24 @@ class CitizenProjectManager
     ];
 
     private $registry;
+    private $storage;
+    private $glide;
 
     public function __construct(ManagerRegistry $registry)
     {
         $this->registry = $registry;
     }
 
+    public function setStorage(Filesystem $storage): void
```

Comment:
As discussed with @Remg, these dependencies should be injected with the constructor.